src/content/docs/en/guides/environment-variables.mdx

```diff
@@ -62,25 +62,21 @@ You can also add environment variables as you run your project:
 <PackageManagerTabs>
  <Fragment slot="yarn">
     ```shell
-    POKEAPI=https://pokeapi.co/api/v2 yarn run dev
+    PUBLIC_POKEAPI=https://pokeapi.co/api/v2 yarn run dev
     ```
  </Fragment>
  <Fragment slot="npm">
     ```shell
-    POKEAPI=https://pokeapi.co/api/v2 npm run dev
+    PUBLIC_POKEAPI=https://pokeapi.co/api/v2 npm run dev
     ```
  </Fragment>
  <Fragment slot="pnpm">
     ```shell
-    POKEAPI=https://pokeapi.co/api/v2 pnpm run dev
+    PUBLIC_POKEAPI=https://pokeapi.co/api/v2 pnpm run dev
```

Comment:
Its from a complete newb point that `PUBLIC_POKEAPI=https://pokeapi.co/api/v2 pnpm run dev` gives an error on terminal since they are not connected with the `&&` operator and that's what I did for about 3-4 times before realizing what's wrong.
Besides that the change is 👍

Reply:
What operating system and terminal are you using? This is the standard way to pass a one time environment variable to a command on Linux, I'm not sure about Mac but I imagine it would be the same. https://stackoverflow.com/a/26189721

If you are using powershell on Windows then the only way to do this is:

```powershell
$env:PUBLIC_POKEAPI="https://pokeapi.co/api/v2"
pnpm run dev
```

and in cmd:

```shell
set PUBLIC_POKEAPI=https://pokeapi.co/api/v2
pnpm run dev
```

Reply:
```shell
$env:PUBLIC_POKEAPI="https://pokeapi.co/api/v2"
pnpm run dev
```
⏫ works smoothly 💯
<hr/>
Pwsh user this side as well, I just copy pasted the snippet and found the mentioned error.

This is a starlight repo but the result is same as in any astro repo

| Command| Output on Terminal|
| ----------- | ----------- |
| PUBLIC_POKEAPI=https://pokeapi.co/api/v2 pnpm run dev| ![image](https://github.com/withastro/docs/assets/88548999/a9319d07-fe7e-4f81-9ff3-33dbc17b8682)|
| pnpm run dev && PUBLIC_POKEAPI=https://pokeapi.co/api/v2| ![image](https://github.com/withastro/docs/assets/88548999/3ea9bdf0-990d-4f20-936c-e6db3b325ec6)        |

Reply:
Hi @Genteure @at-the-vr ! I'm just following up on this conversation here and checking that you're both happy with the proposed changes in this PR?

I tend to let conversations happen and wait to see that they are "resolved" or some very clear signal that everyone is happy before jumping in. So in future, if you're both happy with the outcome of this exchange, there's a "resolve conversation" button here you can hit that collapses the convo and tells me that it's come to a resolution!

I'd like to merge, but want to make sure this thread here actually is resolved, and everyone's happy!

Reply:
Yep I'll mark this as resolved.
Maybe we can add an additional example for Windows users but that can happen in a separate PR.